src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -1572,11 +1573,11 @@ private IEnumerable<string> BuildPackage()
                 Package.Description = Description;
                 Package.Group = Group;
                 Package.Keywords = KeywordList;
-                Package.License = License;
+                Package.License = string.IsNullOrEmpty(License) ? defaultLicense : License;
                 Package.SiteUrl = SiteUrl;
                 Package.RepositoryUrl = RepositoryUrl;
-                Package.CopyrightHolder = CopyrightHolder;
-                Package.CopyrightYear = CopyrightYear;
+                Package.CopyrightHolder = string.IsNullOrEmpty(CopyrightHolder) ? dynamoViewModel.Model.AuthenticationManager.Username : CopyrightHolder;
```

Comment:
hmm... I think there should be some more null checking here, is it possible for AuthenticationManager to be null? Consider that we may want to let sandbox publish locally.

Reply:
hmm I though AuthenticationManager will be BasicAuthManager in Sandbox

Reply:
@mjkkirschner added null check just in case